src/program/lwaftr/tests/subcommands/monitor_test.py

```diff
@@ -12,6 +12,7 @@
 from test_env import DATA_DIR, SNABB_CMD, BaseTestCase, nic_names
 
 
+DAEMON_PROC_NAME = 'monitor_test_daemon'
```

Comment:
As a style note in snabb, can we prefer dashes where possible?  So monitor-test-daemon.